dcos/metronome.py

```diff
@@ -99,6 +112,17 @@ def get_jobs(self):
         response = self._rpc.http_req(http.get, 'v1/jobs')
         return response.json()
 
+    def get_jobs_history(self):
```

Comment:
would it make sense to add a `get_jobs_historySummary`, or making the embeds a function param?